stable/sonatype-nexus/templates/proxy-svc.yaml

```diff
@@ -27,4 +27,9 @@ spec:
   selector:
     app: {{ template "nexus.name" . }}
     release: {{ .Release.Name }}
-  type: {{ .Values.nexus.serviceType }}
+  type: {{ .Values.nexus.service.type }}
+  {{- if eq .Values.nexus.service.type "ClusterIP" }}
+  {{- if .Values.nexus.service.clusterIP }}
+  clusterIP: {{ .Values.nexus.service.clusterIP }}
+  {{- end }}
+  {{- end }}
```

Comment:
You can combine the two if blocks with a single `if and` statement

```yaml
  {{- if and (eq .Values.nexus.service.type "ClusterIP") .Values.nexus.service.clusterIP }}
  clusterIP: {{ .Values.nexus.service.clusterIP }}
  {{- end }}
```

Reply:
fixed